.codegen/_openapi_sha

```diff
@@ -1 +1 @@
-0a1949ba96f71680dad30e06973eaae85b1307bb
\ No newline at end of file
+sha:fbdd0fa3e83fed2c798a58d376529bdb1285b915
```

Comment:
This will require backwards compat in the generator for this format. I recommend not checking this in or the next release will need manual work to fix/override this first.